Makefile

```
@@ -29,5 +29,5 @@ clean:
 serve:
 	pipenv run mkdocs serve
 
-deploy:
-	pipenv run mkdocs gh-deploy --force
+deploy: clean install build
```

```suggestion
deploy:
```
we don't want all those targets here, as it duplicates build on ci